node_virsh.sh

```diff
@@ -6,7 +6,8 @@ DIRECTIVE=${1-}
 
 set -u
 
-NODESXML=`ls node*.xml`
+CURRENT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
CURRENT_DIRECTORY is a misnomer, it's actually the execution directory. Are you trying to avoid the requirement to chdir to $FAME_OUTDIR before running node_virsh.sh?